package.json

```diff
@@ -50,7 +50,7 @@
     "govuk-frontend": "^4.6.0",
     "humanize-string": "^3.0.0",
     "js-cookie": "^3.0.5",
-    "maplibre-gl": "^3.1.0",
+    "maplibre-gl": "^3.3.1",
```

Comment:
Not strictly part of this change and can revert / do separately if useful, but while trying to figure out `setStyle` weirdness, I upgraded. From the changelog, nothing looks relevant for us, and can't hurt to be modern